### ISSUE TITLE
Multilanguage support

### ISSUE THREAD

**Issue by gabn88** [dokterbob/django-newsletter#160]
I have thought about implementing multilanguage support. 

I used to make (:numberOfLanguages) folders, a.k. en_message, de_message, etc. and send it to the user based on the language settings in user.profile.language.

What I will do now, is create a newsletter for each language I have and add users to the list based on the language. Also will catch the post_save of the Profile model to detect changes in language and reflect this in the list the user is in. However, this feels a bit awkward.

**Comment by dokterbob** [comment]
In the applications where multilingual newsletters are required this is indeed the way we do it. This is a method that came to be under a rather thight deadline and, at least from my side, has so far not received much thought. Hence, there might be better ways.

If you have any suggestions for implementing this more elegantly, with a minimal increase in complexity, I would love to hear it. However, there should be a smooth path for transition from the implementation you just described.

Alternately, it would be a welcome addition to the documentation to just simply describe the mentioned approach, especially the `Profile` `save()` hook.

**Comment by gabn88** [comment]
Wow,

I'm having a hard time to understand the django-newsletter with multilanguagal support!

I have created 3 newsletters, same name, different slug (a.k. -en, -nl, and -de).

I have created 3 messages, different name, different slug (a.k. -en, -nl, and -de, but also different name) and attached each one to the corresponsing newsletter. Also attached the right users to the right newsletter.

Now let's start with English. Click 'preview', and then add submission. Automatically checks the right people. So far so good. Click send.

Go to add submission. Click the next message in row, it nicely selects the right addresses. Click save (just to check before sending). It did save the wrong message! Reclick add submission, again select the correct message, again save. Now it has the right message!

Third language, same problem. The first time it selects the wrong message...
